src/trustedtypes.js

```diff
@@ -208,6 +255,12 @@ export const TrustedTypes = (function() {
       if (!policy) {
         throw new Error('Policy not found');
       }
+      // TODO(msamuel): can we require that functionName
+      // is an own property name in policy?
```

Comment:
It should not be required, we only call `buildType..` with constants at initialization of the API, `functionName` is not attacker-controlled.

Reply:
A functionName need not be attacker controlled if it's not present on every instance and there's a prototype.  I think we're probably safe, but hasOwnProperty is cheap and this is policy application code, so I added it just to be safe.  Happy to rollback if that's a problem.